test/testCircuit.js

```diff
@@ -0,0 +1,102 @@
+const path = require('path')
+const { assert } = require('chai')
+const compiler = require('circom')
+const { stringifyBigInts, unstringifyBigInts } = require('../_build/utils/helpers')
+const { Circuit } = require('snarkjs')
+const { hashLeftRight } = require('../_build/utils/crypto')
```

Comment:
I'm not familiar with Flow but I'm not sure that we should import from`_build` rather than from the original source file.

Reply:
Unfortuntately we mocha doesn't read flow syntax and will throw an error if you import it from 'app' :|